.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+style = defaultWithAlign
```

Comment:
@eed3si9n Eugene, is there any chance we could have vertical alignment given the fact that both git and GitHub support ignoring whitespace?
  
Perhaps we could say in the CONTRIBUTING guide that this option should be enabled to browse through the git history? I think this is a good compromise.